src/Features/LanguageServer/Protocol/Features/Options/SymbolSearchOptionsStorage.cs

```diff
@@ -16,7 +17,13 @@ internal static SymbolSearchOptions GetSymbolSearchOptions(this IGlobalOptionSer
             };
 
         public static PerLanguageOption2<bool> SearchReferenceAssemblies =
-            new("dotnet_search_reference_assemblies", SymbolSearchOptions.Default.SearchReferenceAssemblies);
+            new("dotnet_search_reference_assemblies",
+                SymbolSearchOptions.Default.SearchReferenceAssemblies,
+                group: new OptionGroup(FeaturesResources.Add_Imports,
```

Comment:
>Add_Imports [](http://example.com/codeflow?start=57&length=11)

Why "AddImports"? Shouldn't it be something like "SymbolSearch"?

Reply:
It needs to match the description in client. (Let's chat offline about this)